plugins/catalog-backend/src/ingestion/processors/ldap/client.ts

```diff
@@ -62,6 +75,10 @@ export class LdapClient {
       return await new Promise<SearchEntry[]>((resolve, reject) => {
         const output: SearchEntry[] = [];
 
+        this.client.on('error', (err: ldap.Error) => {
```

Comment:
There's no guarantee that this error has anything to do with the existing query right? Might be better to skip this? I'm assuming the `search` callback always gets called, and the error passed in there is the one to use.

Reply:
Let's remove this one actually